File: Assignments/Assignment_15/p6.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int space=0,num=row;
        for(int i=1;i<=row;i++){

                for(int j=1;j<=row;j++){

                        if(space>=j){

                                printf("\t");
                        }else{

                                printf("%d\t",num);
                        }
                }printf("\n\n");
                num--;
                space++;
        }
}

File: JAVA/12.Space_Triangle/p6.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows:");
		int row=sc.nextInt();

		int space=0;

		for(int i=row;i>0;i--){
		
			for(int j=0;j<row;j++){
			
				if(j<space){
				
					System.out.print("\t");
				}else{
				
					System.out.print(i+"\t");
				}
				
			}space++;
			System.out.println();
		}
	}
}
